modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -2968,7 +2968,11 @@ definition {
 
 			ObjectAdmin.goToLayoutsTab();
 
-			WaitForElementPresent(locator1 = "ObjectAdmin#KEBAB_MENU");
+			while ((IsElementNotPresent(locator1 = "ObjectAdmin#KEBAB_MENU")) && (maxIterations = "5")) {
+				Refresh();
+
+				ObjectAdmin.goToLayoutsTab();
+			}
```

Comment:
From the [error](https://testray.liferay.com/reports/production/logs/2023-05/test-1-31/test-portal-testsuite-upstream(master)/2225/functional-tomcat90-mysql57-jdk8/3/6/LocalFile.CreateObject_CanRelateManyOtherEntriesBothObjects/index.html.gz?authuser=0) I saw in the CI, it's as if the click had worked superficially or as if there wasn't enough time for the kebab to appear even with the WaitForElementPresent function. So this loop is an attempt to fix both possible problems.